### ISSUE TITLE
Usability issue. How it works. Progress indicator not contrasted enough

### ISSUE THREAD

**Issue by helloantoine** [covid-alert-ny/covid-green-app#351]
**Issue:** Current progress indicator in the how it works screens is not contrasted enough. Sorry. we must update the style of the progress indicators to pass AA standards of contrast!

**Solution:** 
Inactive bubbles: Make the grey darker #5F5F5F
Active bubble: Same style, BUT make the active bubble 16x16px, instead of 12x12
https://zpl.io/aNXJw9Z

@ahurworth

**Comment by floridemai** [comment]
@helloantoine This should be fixed in the next build.

**Comment by helloantoine** [comment]
Fixed. Well done Paul.
